misc/extension_api_validation/4.1-stable.expected

```diff
@@ -305,3 +305,10 @@ Validate extension JSON: Error: Field 'classes/LineEdit/properties/right_icon':
 Validate extension JSON: Error: Field 'classes/Sprite3D/properties/texture': type changed value in new API, from "Texture" to "Texture2D".
 
 Changed the hint types of some nodes' properties from Texture to Texture2D. No adjustments should be necessary.
+
+
+GH-84419
+--------
+Validate extension JSON: API was removed: classes/Node/constants/NOTIFICATION_NODE_RECACHE_REQUESTED
+
+Remove unused NOTIFICATION_NODE_RECACHE_REQUESTED notification
```

Comment:
Maybe additionally mention that it actively caused a conflict? (which is imo the better reason to break compat than being unused)

Reply:
Are these comments used anywhere? There is already a link to the PR which explains the context.

Reply:
Not sure, but I guess if they weren't, we could just post the link to the PR without description.

Anyway, it's a minor thing, thanks for the change! 👍